public/js/main.js

```diff
@@ -10,7 +10,7 @@ var fpLoaded = function (fp) {
     };
     $.getJSON("/paywall", query, function (res) {
       if(res.enabled){
-        $("#paywall-modal").modal({ keyboard: false, show: true, backdrop: "static" });
+        window.location.replace('https://dashboard.fingerprintjs.com/signup/?utm_source=paywall');
```

Comment:
what if user already logged in?

Reply:
it shouuldn't be a /signup, it should just link to / @kalexey89